src/backend/distributed/utils/metadata_cache.c

```diff
@@ -1452,6 +1455,28 @@ CitusExtraDataContainerFuncId(void)
 }
 
 
+/* return oid of the worker_hash function */
+Oid
+CitusWorkerHashFunctionId(void)
+{
+	Oid citusExtensionOid = InvalidOid;
+	Oid citusSchemaOid = InvalidOid;
+	char *citusSchemaName = 0;
```

Comment:
Actually, we can get these declarations into if block (they are only used there) and merge them with definitions.